components/credit-card/CreditDebitInfoData.tsx

```diff
@@ -1,65 +1,11 @@
-interface WordBank {
-  category: string,
-  answer: string,
-  word: string,
-}
-
-
-export const wordBankData: WordBank[] = [
-  {
-    category: "credit",
-    answer: "borrow",
-    word: "lend",
-  },
-  {
-    category:"credit",
-    answer: "pay",
-    word: "earn",
-  },
-  {
-    category:"debit",
-    answer: "spend",
-    word: "save",
-  },
-  {
-    category:"debit",
-    answer: "deposited",
-    word: "withdraw",
-  },
-  {
-    category:"both",
-    answer: "purchases",
-    word: "sell",
-  },
-  {
-    category:"both",
-    answer: "lost",
-    word: "find",
-  },
-  {
-    category:"both",
-    answer: "stolen",
-    word: "lend",
-  },
-  {
-    category:"both",
-    answer: "plastic",
-    word: "metal",
-  },
-  {
-    category:"both",
-    answer: "magnetic",
-    word: "shiny",
-  },
-  {
-    category:"both",
-    answer: "name",
-    word: "age",
-  },
-  {
-    category:"both",
-    answer: "expiration",
-    word: "postal code",
-  },
-
-]
+export const borrowArray = ["borrow", "lend", "give"]
```

Comment:
move this to the api folder please. the components folder is just for react components